### ISSUE TITLE
Failure when extending a class that includes behavior mixins

### ISSUE THREAD

**Issue by coreyfarrell** [Polymer/polymer#4437]
### Description
Extending a base class which extends Polymer.mixinBehaviors results in failure (at least with iron-a11y-key-behaviors).

#### Live Demo
I could not get jsfiddle / others to work with Polymer 2.0 so I had to create a gh-pages repo.
View: https://coreyfarrell.github.io/broken-test/
Source: https://github.com/coreyfarrell/broken-test/

#### Steps to Reproduce
See index.html from broken-test repo for minimal test case.
When loading the page, press "enter" within the first edit box shown.  This will attempt to create the second edit box, which will fail.  Check the error console for details.

#### Expected Results
Creating my-element1 and my-element2 should work without error.

#### Actual Results
The first created element functions, the second throws an error.  Creating the same element multiple times works fine, for example:
```<my-element1></my-element1><my-element1></my-element1>```

Creating both my-element1 and my-element2 will only succeed on the first element created.  Any further elements that are created will not call `registered` in the behavior, thus causing iron-a11y-key-behavior to fail when attached is run.

The order of declaration for the elements makes no difference, only the first class to have an instance created will succeed.

### Browsers Tested/Affected
- [x] Chrome
- [x] Firefox

### Versions
- Polymer: v2.0.0-rc.3
- webcomponents: v1.0.0-rc.6

**Comment by sorvell** [comment]
Yes, can confirm this issue. It's related to how we call `registered` when extending classes. We'll work on a fix.
